src/NumericalAlgorithms/Integration/GslQuadAdaptive.hpp

```diff
@@ -50,6 +47,15 @@ double integrand(double x, void* params) {
                                                      args->args_);
 }
 
+/*!
```

Comment:
remove `!` since this won't show up in Doxygen anyway. Also I think we prefer the `//`-style for these kind of explanatory comments but that's really minor and I don't request you change that